samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md

```diff
@@ -0,0 +1,127 @@
+
+## Prerequisites
+
+* A pulsar broker running in any mode - TLS,JWT or plain.
+* Depending on the mode and where your broker is running, you will have to configure the pulsar connection
+
+
+## Import the sample
+
+1. Download the sample's .json file 'PulsarChunkingApp.json','PulsarBatchingProducer.json' and 'PulsarBatchingConsumer.json'
+
+2. Create a new empty app.
+![Create an app](../../../../import-screenshots/2.png)
+
+3. On the app details page, select Import app.
+![Select import](../../../../import-screenshots/3.png)
+
+4. Browse on your machine or drag and drop the .json file for the app that you want to import.
+![Import your sample](../../../../import-screenshots/pulsar/ImportApp.png)
+
+5. Click Upload. The Import app dialog displays some general warnings and specific error in you app if any.
+![The Import app dialog](../../../../import-screenshots/pulsar/ImportDialog.png)
+
+6. You have the option to import all flows from the source app or selectively import flows.
+
+7. If you choose selective import, select the trigger, flow and connection. Click Next.
+
+8. Make sure you re-configure the connection as mentioned in 'Understanding the configuration' section
+
+## Understanding the configuration
+
+### The Connection
+When you import this app, you need to re-configure the 'pulsarConnection' connection in Connections page. 
+
+![The pulsar connection](../../../../import-screenshots/pulsar/pulsar_connection.png)
+
+
+In the sample connection,
+* It uses a pulsar broker running TLS mode. You can set the authorization type according to your broker flavour. If its running in plain mode, simply choose 'None' in 'auth type' dropdown.
+* In TLS or JWT auth types, broker URL requires 'pulsar+ssl' in the beginning of the URL. In plain mode, only 'pulsar' is required.
+* Update the Broker URL and certificates in the connection if using TLS mode. Refer to Pulsar Security documentation to get the idea about auth types and creating certificates- https://pulsar.apache.org/docs/en/security-overview  
+For more information on connection fields, please refer to https://integration.cloud.tibco.com/docs/index.html#Subsystems/flogo-pulsar/connector-users-guide/pulsar-connection-details.html
+* Connecting to Pulsar hosted on Tibco Cloud Messaging - You can also connect to the pulsar TCM instance by configuring Pulsar Connection using tcm-config.yaml. In this case you need to use Autorization type as JWT and use the tcm_authentication_key as the JWT token. You can get the broker url and namespace from the same tcm-config.yaml.
+Please refer to - https://messaging.cloud.tibco.com/docs/getstarted/quickstart-pulsar/index.html
+
+### The Producer and Consumer Flows
```

Comment:
As you are using 3 applications. Please give proper heading for all 3 app name and then description about flow.